src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,8 @@
 // Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
+  <>
```

Comment:
There is no need to use fragments while you are rendering just one element inside